### PR TITLE
doc: add snapshot/windows driver selection in helm chart installation

### PR DESCRIPTION
**Pull request by ZeroMagic**
**What type of PR is this?**
/kind documentation
/kind flake

**What this PR does / why we need it**:

- add snapshot/windows driver selection in helm chart installation
- add the latest helm chart configuration
- update verify-helm-chart.sh 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
NONE
```

### CODE REVIEW

**Comment by ZeroMagic** [comment]
It works now. PTAL @andyzhangx

File: charts/README.md [hunk 1]
```diff
@@ -19,3 +19,52 @@ $ helm install azuredisk-csi-driver azuredisk-csi-driver-latest.tgz --namespace
...
+| `image.nodeDriverRegistrar.pullPolicy`            | csi-node-driver-registrar image pull policy                | IfNotPresent                                                 |
+| `serviceAccount.create`                           | whether create service account of csi-azuredisk-controller | true                                                         |
+| `rbac.create`                                     | whether create rbac of csi-azuredisk-controller            | true                                                         |
+| `controller.replicas`                             | the replicas of csi-azuredisk-controller                   | 2                                                            |
+| `node.metricsPort`                                | the metrics port of node                                   | 29605                                                        |
+| `snapshot.create`                                 | whether enable snapshot feature                            | true                                                         |
```
Comment (andyzhangx) [inline charts/README.md]: should be false by default, otherwise azure disk driver would depends on 1.17

Comment (andyzhangx) [reply charts/README.md]: `snapshot.enabled`

File: charts/README.md [hunk 2]
```diff
@@ -19,3 +19,52 @@ $ helm install azuredisk-csi-driver azuredisk-csi-driver-latest.tgz --namespace
...
+| `snapshot.image.csiSnapshotController.tag`        | snapshot-controller docker image tag                       | v2.0.0                                                       |
+| `snapshot.image.csiSnapshotController.pullPolicy` | snapshot-controller image pull policy                      | IfNotPresent                                                 |
+| `snapshot.snapshotController.replicas`            | the replicas of snapshot-controller                        | 1                                                            |
+| `snapshot.snapshotController.serviceAccount`      | whether create service account of snapshot-controller      | true                                                         |
+| `snapshot.snapshotController.rbac`                | whether create rbac of snapshot-controller                 | true                                                         |
+| `windows.create`                                  | whether enable windows feature                             | false                                                        |
```
Comment (andyzhangx) [inline charts/README.md]: `windows.enabled`

File: charts/README.md (hunk 1 above)
Comment (ZeroMagic) [reply charts/README.md]: done

File: charts/README.md (hunk 2 above)
Comment (ZeroMagic) [reply charts/README.md]: done

**Pull request merged**